admin/lib/consts/service-roles.js

```diff
@@ -0,0 +1,6 @@
+module.exports = Object.freeze({
```

Comment:
This files should just be named roles.js - it should be all roles in the system.

Reply:
@jon-shipley done